### DISCUSSION TITLE
I can't connect to mathjs.org

### DISCUSSION THREAD

**Discussion by qgustavor** [josdejong/mathjs#3313] [Q&A] [answered]
Example:

```shell
$ curl --verbose mathjs.org
* Host mathjs.org:80 was resolved.
* IPv6: (none)
* IPv4: 185.199.108.153, 185.199.110.153, 185.199.111.153, 185.199.109.153
*   Trying 185.199.108.153:80...
* connect to 185.199.108.153 port 80 from 0.0.0.0 port 51205 failed: Timed out
*   Trying 185.199.110.153:80...
* connect to 185.199.110.153 port 80 from 0.0.0.0 port 51215 failed: Timed out
*   Trying 185.199.111.153:80...
* connect to 185.199.111.153 port 80 from 0.0.0.0 port 51231 failed: Timed out
*   Trying 185.199.109.153:80...
* connect to 185.199.109.153 port 80 from 0.0.0.0 port 51241 failed: Timed out
* Failed to connect to mathjs.org port 80 after 84229 ms: Could not connect to server
* closing connection #0
curl: (28) Failed to connect to mathjs.org port 80 after 84229 ms: Could not connect to server
```

I noticed it's hosted on GitHub Pages. My website (qgustavor.tk) is also hosted on GitHub Pages, so I compared the results of connecting to it with the results when connecting to mathjs.org: looks like a IPv6 address is missing from the DNS configuration (and my ISP is currently not providing IPv4 access for whatever reason). Sure, is a apex domain, that's harder to set up, in my case I'm using Cloudflare's CNAME flattening, but maybe that's not available.

Anyway, I just want to report that. Maybe there are other users affected by this. I will try to workaround this issue setting up the GitHub Pages's IPv6 addresses for mathjs.org on my DNS server.

**Comment by josdejong** [comment, marked as answer]
Thanks for reporting! IPv6 wasn't a thing when I did set up the website 11 years ago 😅 .

I've added DNS for IPv6 now:

```bash
$ curl --verbose mathjs.org
* Host mathjs.org:80 was resolved.
* IPv6: 2606:50c0:8002::153, 2606:50c0:8003::153, 2606:50c0:8000::153, 2606:50c0:8001::153
* IPv4: 185.199.110.153, 185.199.109.153, 185.199.111.153, 185.199.108.153
```
